packages/engine/tests/units/globals/mod.rs

```diff
@@ -12,6 +12,6 @@ mod py {
 
     run_test!(access_from_behaviors, Python);
     run_test!(access_from_init, Python);
-    run_test!(immutable_from_behaviors, Python);
-    run_test!(immutable_from_init, Python);
+    run_test!(immutable_from_behaviors, Python, #[ignore]); // TODO
+    run_test!(immutable_from_init, Python, #[ignore]); // TODO
```

Comment:
Could you elaborate on the TODO (like in other tests)?

Reply:
```suggestion
    // TODO - tests disabled until we make it so that the data actually is immutable
    run_test!(immutable_from_behaviors, Python, #[ignore]);
    run_test!(immutable_from_init, Python, #[ignore]);
```